criu/cgroup.c

```diff
@@ -586,10 +668,17 @@ static int collect_cgroups(struct list_head *ctls)
 		if (!opts.manage_cgroups)
 			continue;
 
+		snprintf(mopts, sizeof(mopts), "xattr,%s", cc->name);
```

Comment:
Pls write a comment why we need to care about xattr

Reply:
done.